test/quantization/test_quant_api.py

```diff
@@ -268,57 +268,6 @@ def test_8da4w_quantizer_eval(self):
             f"accuracy regressed from 8.23 to {result['results']['wikitext']['word_perplexity,none']}"
         )
 
-    @unittest.skip("skipping until we get checkpoints for gpt-fast")
-    def test_gptq_quantizer_gpt_fast(self):
```

Comment:
is this test not useful to keep around or some new version of it?

Reply:
oh next test tests gpt fast code path: test_gptq_quantizer_int4wo, this was initially added because we were trying to merge the gpt fast code path and 8da4w code path together in the same quantizer code path and use a flag to distinguish them, but now we just duplicated the quantizer code